src/compute/src/row_spine.rs

```diff
@@ -117,35 +120,23 @@ mod container {
         }
     }
 
-    impl Default for DatumContainer {
-        fn default() -> Self {
-            Self {
-                batches: Vec::new(),
-            }
-        }
-    }
-
     impl BatchContainer for DatumContainer {
         type PushItem = Row;
         type ReadItem<'a> = DatumSeq<'a>;
 
         fn push(&mut self, item: Self::PushItem) {
             self.copy_push(&item);
         }
+
         fn copy_push(&mut self, item: &Self::PushItem) {
             self.copy(MyTrait::borrow_as(item));
         }
```

Comment:
I think you can remove these if you like.

Reply:
Removed.